test/mason/search/no-pattern.good

```diff
@@ -1,3 +1,3 @@
-WithCaps (0.1.0)
 capsize (0.1.0)
```

Comment:
I got this order in testing:

```
multiVersion (1.0.0)
WithCaps (0.1.0)
capsize (0.1.0)
```

Did you see this on your end?

Reply:
@ben-albrecht I don't get this on my end.

Reply:
Investigated a bit further -- see https://github.com/chapel-lang/chapel/pull/15789/files#r437742648